dts/arm/st/l4/stm32l471-pinctrl.dtsi

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/stm32-pinctrl.h>
+
+/ {
+	soc {
+		pinctrl: pin-controller@48000000 {
```

Comment:
Could add a

    compatible = "st,stm32-pinmux.yaml"

here now to try it out.

Reply:
Actually, this node is already defined (with `compatible = "st,stm32-pinmux";`) in `dts/arm/st/l4/stm32l4.dtsi`.
Le's keep the compatible name for now (used in all sm32 dts files). We'll see later if this needs renaming.